knox-homepage-ui/angular-cli.json

```diff
@@ -0,0 +1,56 @@
+{
+  "project": {
+    "version": "1.0.0-beta.22-1",
+    "name": "ng-knox-homepage"
+  },
+  "apps": [
+    {
+      "root": "homepage",
+      "outDir": "target/classes/homepage/app",
+      "assets": [
+        "assets",
+        "favicon.ico"
+      ],
+      "index": "index.html",
+      "main": "main.ts",
+      "test": "test.ts",
+      "tsconfig": "tsconfig.json",
+      "prefix": "app",
+      "mobile": false,
+      "styles": [
+        "../node_modules/bootstrap/dist/css/bootstrap.min.css",
+        "styles.css"
+      ],
+      "scripts": [
+        "../node_modules/jquery/dist/jquery.min.js",
+        "../node_modules/bootstrap/dist/js/bootstrap.js",
+        "../node_modules/ace-builds/src-min/ace.js",
```

Comment:
Do we need `ace` since I think thats the editor.

Reply:
Removed all that not needed.